pkg/cmd/workloads.go

```diff
@@ -39,3 +41,17 @@ func printWorkloadList(workloadList []types.Capability, ioStreams cmdutil.IOStre
 	ioStreams.Info(table.String())
 	return nil
 }
+
+func InstalledWorkloads() (string, error) {
```

Comment:
I believe there is a similar function located in `pkg/plugins/local.go`. Would you please check it first?